$OpenBSD: patch-uigtk_ml,v 1.2 2005/05/30 19:27:45 sturm Exp $
Post release port maintainer fix.
o Fix compilation with GTK 1.
  Applied the same approach used in beta version by developers.
  Approved by developers.
  See http://lists.seas.upenn.edu/pipermail/unison-hackers/2005-May/000074.html
--- uigtk.ml.orig	Mon Sep  6 15:15:46 2004
+++ uigtk.ml	Tue May 17 02:21:59 2005
@@ -1606,7 +1606,7 @@ in
                   catch (fun () ->
                            Transport.transportItem
                              theSI.ri (Uutil.File.ofLine i)
-                             (fun title text -> Trace.status (Printf.sprintf "\n%s\n\n%s\n\n" title text))
+                             (fun title text -> Trace.status (Printf.sprintf "\n%s\n\n%s\n\n" title text); true)
                            >>= (fun () ->
                            return Util.Succeeded))
                         (fun e ->
@@ -1760,6 +1760,7 @@ in
   actionBar#insert_space ();
   grAdd grDiff (actionBar#insert_button ~text:"Diff" ~callback:diffCmd ());
 
+(*
   let mergeCmd () =
     match !current with
       Some i ->
@@ -1787,7 +1788,7 @@ in
 
   actionBar#insert_space ();
   grAdd grDiff (actionBar#insert_button ~text:"Merge" ~callback:mergeCmd ());
-
+*)
   (*********************************************************************
     Keyboard commands
    *********************************************************************)
